Headlines/src/Controllers/TrendingNewsTableViewController.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  TrendingNewsTableViewController.swift
+//  Canillitapp
+//
+//  Created by Ezequiel Becerra on 08/12/2018.
+//  Copyright © 2018 Ezequiel Becerra. All rights reserved.
+//
+
+import UIKit
+
+class TrendingNewsTableViewController: NewsTableViewController {
+
+    var headerViewModel: TopicHeaderViewModel?
+    
+    let headerViewHeight: CGFloat = 180.0
+    var headerViewHeightConstraint: NSLayoutConstraint?
+    var headerViewTopConstraint: NSLayoutConstraint?
+    
+    private func setupTopicHeaderView() {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/962ff671bf260eb1c01ac4072725c47d0275e505/Headlines/src/Controllers/TrendingNewsTableViewController.swift#L19"></span>Function body should span 40 lines or less excluding comments and whitespace: currently spans 54 lines</td>
    </tr>
  </tbody>
</table>